src/conn/conn_log.c

```diff
@@ -254,6 +254,11 @@ __logmgr_config(
 		WT_RET(__wt_compressor_config(
 		    session, &cval, &conn->log_compressor));
 
+		WT_RET(__wt_config_gets_none(
```

Comment:
I would add a comment that this code must be called after the above code so that the `log_compressor` is known to be set up.